target/linux/armsr/armv7/config-6.1

```diff
@@ -1,17 +1,25 @@
+# CONFIG_AHCI_ST is not set
 CONFIG_ALIGNMENT_TRAP=y
 CONFIG_ARCH_32BIT_OFF_T=y
 CONFIG_ARCH_MIGHT_HAVE_PC_PARPORT=y
+CONFIG_ARCH_MMAP_RND_BITS=8
 CONFIG_ARCH_MULTIPLATFORM=y
 CONFIG_ARCH_MULTI_V6_V7=y
 CONFIG_ARCH_MULTI_V7=y
-CONFIG_ARCH_MMAP_RND_BITS=8
 CONFIG_ARCH_NR_GPIO=0
 CONFIG_ARCH_OPTIONAL_KERNEL_RWX=y
 CONFIG_ARCH_OPTIONAL_KERNEL_RWX_DEFAULT=y
 CONFIG_ARCH_SELECT_MEMORY_MODEL=y
 CONFIG_ARCH_VIRT=y
+CONFIG_ARCH_STM32=y
+CONFIG_ARM_GLOBAL_TIMER=y
+CONFIG_ARM_GT_INITIAL_PRESCALER_VAL=1
 CONFIG_ARM=y
 CONFIG_ARM_CPU_SUSPEND=y
+CONFIG_ARM_ERRATA_754322=y
```

Comment:
Why are you selecteding this errata? It is selected by STI but not STM32.

Reply:
Removed. Thanks!